base/applications/notepad/lang/ro-RO.rc

```diff
@@ -161,18 +161,18 @@ BEGIN
     STRING_UNTITLED "Fără titlu"
     STRING_ALL_FILES "Orice fișier (*.*)"
     STRING_TEXT_FILES_TXT "Fișiere text (*.txt)"
-    STRING_TOOLARGE "Fișierul „%s” este prea mare.\n\
-Încercați un alt editor."
-    STRING_NOTEXT "Nu ați scris nimic.\
-\nScrieți ceva apoi încercați din nou."
-    STRING_DOESNOTEXIST "Fișierul „%s”\nnu există.\n\n\
-Creați un nou fișier?"
+    STRING_TOOLARGE "Fișierul „%s” este prea lung pentru Notepad...\n\
+Utilizaţi alt editor pentru acest fişier."
+    STRING_NOTEXT "Nu ați scris nimic.\n\
+Scrieți ceva apoi încercați din nou."
+    STRING_DOESNOTEXIST "Fișierul „%s”\n\nu există.\n\n\
```

Comment:
you changed the text from 1 newline plus "nu există"
to 2 newlines plus "u există".
Was that intentional or a typo?

Reply:
Bht \n\ doesn't mean a new line? I didn't want to do any typo.

Reply:
I fixed it. I didn't understand the rules very well.

Reply:
Can happen. That's what a review is for. Resolved.